src/goto-instrument/full_slicer.cpp

```diff
@@ -60,6 +60,10 @@ void full_slicert::add_function_calls(
   queuet &queue,
   const goto_functionst &goto_functions)
 {
+  // do not retain calls of empty functions
+  if(node.PC->function.empty())
```

Comment:
I'm not sure whether the comment is misleading or the code might not do what you'd expect it to do? This tests whether the function name set for a particular instruction is empty, which is unrelated to the function being called not having a function body.